src/styles/partials/components/_page-header.scss

```diff
@@ -0,0 +1,40 @@
+.dg_page-header {
+  //background-image: url(/sebin/r/t/prototype-subpage-landing-header-bkg.jpg);
```

Comment:
We still need this functionality, but the image needs to be part of the component instead of the script. The reason for this is that web services will be managing what these look like.

This is the full path for the image to use. https://www.baltimorecountymd.gov/sebin/r/t/prototype-subpage-landing-header-bkg.jpg

Reply:
please let me know if you have any questions.

Reply:
done